app/resources/views/pages/location_report/collapsed.blade.php

```diff
@@ -22,8 +22,9 @@
 		'location' => $location))
 	<div class="basic-info">
 		<div class="addresses">
-			<address>{{ $location->address }}</address>
-			<a href="{{ $location->getExternalWebURL() }}">{{ $location->getExternalWebURL() }}</a>
+		<p><em class="fa fa-map-marker"></em>{{ $location->address }}</p>
+		@if(!is_null($location->phone_number))<p><em class="fa fa-phone"></em>{{ $location->phone_number }}</p>@endif
+		<a href="{{ $location->getExternalWebURL() }}">{{ $location->getExternalWebURL() }}</a>
```

Comment:
Keep the original extra indentation that indicated that the elements inside ```<div class="addresses">``` were an extra level deeper in the document tree.